### PR TITLE
Add a post announcing our CDN on Community

### PR DESCRIPTION
**Pull request by ericholscher**
This post has a few goals:

* Thank CF for their sponsorship
* Tell users the CDN exists and they should be happy
* Explain how it effects our ongoing product roadmap, allowing us to do more things (like redirects)

### CODE REVIEW

File: announcing-community-cdn.rst [hunk 1]
```diff
@@ -0,0 +1,80 @@
+Shipping a CDN on Read the Docs Community
```
Comment (humitos) [inline announcing-community-cdn.rst:1]: Meta tags are missing here (`.. post` with author's name and date, and `.. meta` with SEO info)

File: announcing-community-cdn.rst [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+but CDN is one of the most complicated.
+
+To make a CDN function across all our sites,
+every data center that the CDN has needs to work with all our custom domains.
+Imagine we have 5,000 domains and there are 100 data centers across the world,
+this means that ``5,000 * 100 = 500,000`` different endpoints need to be configured for this to work at scale.
+
+We are lucky that CloudFlare has the global scale to be able to donate us this service.
+Specifically,
+their `SSL for SAAS`_ service is what we're using for both SSL and CDN across all our custom domains
```
Comment (humitos) [inline announcing-community-cdn.rst:29]:
```suggestion
their `SSL for SAAS`_ service is what we're using for both SSL and CDN across all our custom domains.
```

File: announcing-community-cdn.rst [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+This lets us add a ``Cache-Tag`` header to all the documentation that we serve,
+and bust the cache using just that tag.
+
+An example,
+when you load our docs,
+we will return a header::
+
+    GET https://docs.readthedocs.io/en/latest/
+    ...
+    Cache_tag: docs-latest,docs
```
Comment (humitos) [inline announcing-community-cdn.rst:46]:
```suggestion
    Cache-Tag: docs-latest,docs
```

Just consistency with the mention in the previous paragraph.

Comment (humitos) [reply announcing-community-cdn.rst:46]:
```suggestion
    Cache-Tag: docs,docs-latest
```

Sort the values in the same way they are mentioned in the following paragraph.

File: announcing-community-cdn.rst [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+
+We return a tag that matches both the ``$project``, and the ``$project-$version``.
+This allows us to bust the cache for a specific version,
```
Comment (humitos) [inline announcing-community-cdn.rst:49]:
```suggestion
This allows us to invalidate the cache for a specific version,
```

Invalidate sounds more familiar to me when talking about cache.

Comment (humitos) [reply announcing-community-cdn.rst:49]: Hrm... There are more uses of the "bust" word in the next paragraphs. I assume it's just to not repeat yourself too much.

File: announcing-community-cdn.rst [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+As you know,
+cache invalidation is one of the harder problems,
+so we take a pretty conservative approach.
+We bust the cache in the following scenarios:
+
+* Bust Project cache on Project or Domain save
+* Bust Version cache on documentation builds for specific versions
+
+The client code is quite simple,
+a single HTTP request with a list of ``tags`` to clear.
```
Comment (humitos) [inline announcing-community-cdn.rst:61]:
```suggestion
a single HTTP request to Cloudflare's API with a list of ``tags`` to clear.
```

File: announcing-community-cdn.rst [hunk 6]
```diff
@@ -0,0 +1,90 @@
+.. post:: May 12, 2020
+   :tags: community, cdn, hosting
+   :author: Eric
+
+.. meta::
+   :description lang=en:
+
+      Read the Docs Community now has a CDN enabled for all projects.
```
Comment (davidfischer) [inline announcing-community-cdn.rst:8]: This meta description is probably a little too short and probably just won't get used by search engines. It's a pretty [inexact science](https://moz.com/learn/seo/meta-description) whether one gets used but I'd suggest something like this:

```suggestion
      Read the Docs Community now has Cloudflare CDN enabled for all projects resulting in faster and more secure documentation worldwide.
```

**Comment by ericholscher** [comment]
I've prepped this to go out on Monday 👍

**Pull request merged**